ebu_tt_live/bindings/converters/ebutt3_ebuttd.py

```diff
@@ -102,21 +202,25 @@ def convert_styling(self, styling_in, dataset):
         return new_elem
 
     def convert_style(self, style_in, dataset):
-        lineHeight = style_in.lineHeight
-        if lineHeight is not None:
-            if lineHeight.endswith('c'):
-                lineHeight = lineHeight[:-1]+'00%'
+        color = style_in.color
+        if color is not None:
+            if isinstance(color, ebuttdt.namedColorType):
+                color = ebuttdt.named_color_to_rgba(color)
+        backgroundColor = style_in.backgroundColor
+        if backgroundColor is not None:
+            if isinstance(backgroundColor, ebuttdt.namedColorType):
+                backgroundColor = ebuttdt.named_color_to_rgba(backgroundColor)
         new_elem = d_style_type(
             *self.convert_children(style_in, dataset),
             id=style_in.id,
             style=style_in.style,  # there is no ordering requirement in styling so too soon to deconflict here
             direction=style_in.direction,
             fontFamily=style_in.fontFamily,
-            fontSize=None,  #style_in.fontSize TODO: Calculate fonts appropriately
-            lineHeight=lineHeight,
+            fontSize=None,  # It is far easier to regenerate fontSizes at the moment than introspecting the cases
```

Comment:
Well this is the entire reason for the existence of this entire branch... what would you rather I put there?